nova_vm/src/engine/bytecode/executable.rs

```diff
@@ -1313,12 +1313,30 @@ impl CompileEvaluation for ast::StaticMemberExpression<'_> {
             ctx.exe.add_instruction(Instruction::GetValue);
         }
 
+        ctx.exe.add_instruction(Instruction::LoadCopy);
```

Comment:
**issue:** Let's avoid the extra load-store work if we don't have optional. So move the instruction adding inside the if and return a Some(JumpIndex) or None if we don't have conditional access.

Reply:
Done